src/Compatibility/Core/src/iOS/Forms.cs

```diff
@@ -167,14 +168,18 @@ public static void SetFlags(params string[] flags)
 				Profile.Enable();
 		}
 
-		public static void Init()
+		public static void Init(IActivationState activationState)
```

Comment:
Like with Android, we can wrap all the things for the user if nothing was specified.

Reply:
Same comment here as above. I think Init will go away once all is said and done. This is just a way to make it all work for now